rpm/SPECS/openresty.spec

```diff
@@ -164,10 +160,6 @@ mkdir -p %{buildroot}/etc/init.d
 export QA_RPATHS=$[ 0x0002 ]
 
 
-%clean
-rm -rf %{buildroot}
-
```

Comment:
@agentzh In the absence of a "%clean" directory, the default action is to rm -rf buildroot. If you add %clean by itself then buildroot is preserved (good for debugging). It didn't seem necessary to explicitly do the rm -rf.